include/mc_rbdyn/JointSensor.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2015-2022 CNRS-UM LIRMM, CNRS-AIST JRL
+ */
+
+#pragma once
+
+#include <mc_rbdyn/api.h>
+
+#include <limits>
+#include <memory>
+#include <string>
+
+namespace mc_rbdyn
+{
+
+struct JointSensor;
+using JointSensorPtr = std::unique_ptr<JointSensor>;
+
+/** This structure defines a joint sensor that provides
+ * temperature and current information for a specfic joint. */
+struct MC_RBDYN_DLLAPI JointSensor
+{
+  /** Default constructor, does not represent a valid joint sensor */
+  inline JointSensor() : JointSensor("", "") {}
+
+  /** Constructor
+   *
+   * @param name Name of the sensor
+   *
+   * @param jointName Name of the joint to which the sensor is attached
+   *
+   */
+  JointSensor(const std::string & name, const std::string & jointName)
+  {
+    name_ = name;
+    joint_ = jointName;
+    type_ = "JointSensor";
+  }
+
+  /** Constructor
+   *
+   * @param jointName Name of the joint to which the sensor is attached
+   *
+   */
+  JointSensor(const std::string & jointName);
+
+  JointSensor(const JointSensor & js) : JointSensor(js.name(), js.joint())
+  {
+    motor_temperature_ = js.motor_temperature_;
+    driver_temperature_ = js.driver_temperature_;
+    motor_current_ = js.motor_current_;
+  }
+
+  JointSensor & operator=(const JointSensor & js)
+  {
+    name_ = js.name_;
+    joint_ = js.joint_;
+    motor_temperature_ = js.motor_temperature_;
+    driver_temperature_ = js.driver_temperature_;
+    motor_current_ = js.motor_current_;
+    return *this;
+  }
```

Comment:
This should be defaulted as well.

Reply:
I "defaulted" it. But could you please see if this is the correct way to do it.

Reply:
That's the correct way to do it :)